.github/workflows/collab-guard.yml

```
@@ -5,6 +5,58 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0  # Need full history for diff calculation
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)